opentracing/src/main/java/opentracing/Span.java

```diff
@@ -65,29 +63,7 @@
   Span setTag(String key, Number value);
 
   /**
-   * {@code message} is a format string and can refer to fields in the payload by path, like so:
-   *
-   * <pre>{@code
-   *
-   * span.info("first transaction is worth ${transactions[0].amount} ${transactions[0].currency}",
-   *     ImmutableMap.of(
-   *       "transactions", asList(
-   *         Transaction.builder().amount(10).currency("USD").build(),
-   *         Transaction.builder().amount(11).currency("USD").build(),
-   *       )
-   *     )
-   * );
-   * }</pre>
-   *
-   * @param message {@link Formatter format string} that can refer to fields in the args payload.
-   * @param args arbitrary payload
-   */
-  // See https://github.com/opentracing/opentracing.github.io/issues/30 about parameterization
-  Span info(String message, Object... args);
-
-  /** Same as {@link #info}, but for warnings. */
-  Span warning(String message, Object... args);
-
-  /** Same as {@link #info}, but for errors. */
-  Span error(String message, Object... args);
+   * @todo
+   **/
+  Span event(String message, /* @Nullable */ Object payload);
```

Comment:
Ok.

>  I'm also fine only-supporting log for now.

I'll change all the methods to `log`. (Having different names only adds schema and unnecessary overhead to the newcomer).

> Prefer timestamp at position zero

I'll put the timestamp first.

> putting nulls in the middle looks worse

I don't like using nulls either, but that's porcelain api we can easily add later on.

> Instant is the common way of representing timestamps.

I was thinking we will have services running in different timezones and will need to deal with that, but Instant forces us to use UTC time. I like that.